src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -6,14 +6,15 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.List;
 
 public class WorkWithFileTest {
     private WorkWithFile workWithFile = new WorkWithFile();
     private static final String APPLE_RESULT_FILE = "appleResult.csv";
     private static final String GRAPE_RESULT_FILE = "grapeResult.csv";
     private static final String ORANGE_RESULT_FILE = "orangeResult.csv";
     private static final String BANANA_RESULT_FILE = "bananaResult.csv";
-
+    private List<Integer> list;
```

Comment:
before committing make sure you don't make changes to the files you aren't supposed to, like tests